src/services/AdminApiContext.ts

```diff
@@ -89,7 +89,7 @@ export class AdminApiContext {
                 grant_type: 'client_credentials',
                 client_id: options.client_id,
                 client_secret: options.client_secret,
-                scope: ['write'],
+                scope: 'write',
```

Comment:
we need to support older shopware versions, so we should try both variations, if we can't detect the version (I believe that is only possible after auth)

Reply:
I know, it should have always worked to provide plain string therefore there should be no need to switch the behaviour